src/components/try-it/call-response/CallResponse.tsx

```diff
@@ -19,8 +28,13 @@ export default function CallResponse() {
                                 <label className="collapse-title text-xl font-medium break-all break-words" htmlFor={`response-${response.timestamp}`}>
                                     {response.url}
                                 </label>
-                                <div className="collapse-content prose max-w-none">
-                                    <pre>{JSON.stringify(response.data, null, 4)}</pre>
+                                <div className="collapse-content prose max-w-none text-l">
+                                    <CopyButton></CopyButton>
```

Comment:
No children are present, so make this a self-closing element (`<CopyButton />`).